thoth/user_api/api_v1.py

```diff
@@ -563,6 +560,39 @@ def list_buildlogs(page: int = 0):
     return _do_listing(BuildLogsStore, page)
 
 
+def get_info():
+    """Get information about Thoth deployment."""
+    return {
+        "deployment_name": os.getenv("THOTH_DEPLOYMENT_NAME"),
+        "version:": os.getenv("OPENSHIFT_BUILD_REFERENCE", "@dev"),
+        "s3_endpoint_url": os.getenv("THOTH_S3_ENDPOINT_URL"),
+        "dgraph_host": os.getenv("GRAPH_SERVICE_HOST"),
+        "amun_api_url": os.getenv("AMUN_API_URL"),
+        "frontend_namespace": os.getenv("THOTH_FRONTEND_NAMESPACE"),
+        "middletier_namespace": os.getenv("THOTH_MIDDLETIER_NAMESPACE"),
+        "backend_namespace": os.getenv("THOTH_BACKEND_NAMESPACE"),
+        "s3_bucket_prefix": os.getenv("THOTH_CEPH_BUCKET_PREFIX"),
+    }
+
+
+def list_python_packages(page: int = 0):
+    """
+    Retrieve information from the Knowledge Graph.
+
+    This will include Python Package Names and the number
+    of distinct and solved Versions.
+    """
+    graph = GraphDatabase()
+    graph.connect()
+
+    package_versions = [
+        {name: count} for name, count in graph.get_python_package_versions_all_count(
+            count=PAGINATION_SIZE,
+            start_offset=page*PAGINATION_SIZE).items()
+    ]
+    return package_versions
```

Comment:
What about just:


```python
return graph.get_python_package_versions_all_count(
    count=PAGINATION_SIZE,
    start_offset=page*PAGINATION_SIZE
)
```

BTW it would be nice to return pagination information in response headers.